bin/svg-sprite.js

```diff
@@ -44,39 +45,38 @@ var _ = require('lodash'),
  * @return {void}
  */
 function addOption(name, option) {
-	var alias = name;
-
-	// If the this is an option itself
-	if ('description' in option) {
-		if ('alias' in option) {
-			alias = option.alias;
-			yargs = yargs.alias(alias, name);
-		}
-
-		yargs = yargs.describe(alias, option.description);
-
-		if ('default' in option) {
-			var template = name.endsWith('-template'),
-				def = template ? path.resolve(path.dirname(__dirname), option.default) : option.default;
-			yargs = yargs.default(alias, def);
-
-			if ((option.default === true) || (option.default === false)) {
-				yargs = yargs.boolean(name);
-			}
-
-		} else if (option.required) {
-			yargs = yargs.require(alias);
-		}
-
-		if ('map' in option) {
-			map[option.map] = name;
-		}
-	}
-
-	var children = _.omit(option, ['description', 'alias', 'default', 'map']);
-	for (var sub in children) {
-		addOption(name + '-' + sub, children[sub]);
-	}
+    let alias = name;
+
+    // If the this is an option itself
+    if ('description' in option) {
+        if ('alias' in option) {
+            alias = option.alias;
+            yargs = yargs.alias(alias, name);
+        }
+
+        yargs = yargs.describe(alias, option.description);
+
+        if ('default' in option) {
+            const template = name.endsWith('-template');
+            const def = template ? path.resolve(path.dirname(__dirname), option.default) : option.default;
+            yargs = yargs.default(alias, def);
+
+            if (option.default === true || option.default === false) {
+                yargs = yargs.boolean(name);
+            }
+        } else if (option.required) {
+            yargs = yargs.require(alias);
+        }
+
+        if ('map' in option) {
+            optionsMap[option.map] = name;
+        }
+    }
+
+    const { description, alias: optAlias, default: optDefault, map, ...children } = option;
```

Comment:
Im not 1000% sure if I already fully understand what's going on here but I somehow suspect it should be `optionsMap` here instead of `map`. Possible?

Reply:
`map` is the `option.map` key we are ignoring like everything else before that too. We only want the children.

Reply:
Got it, thanks. I was just wondering because there's another `const map` down below in line 138. But maybe I can't fully see the scopes here in the diff view ...

Reply:
Hence why I suggesting viewing each patch separately with the `?w=1` query appended :)

Reply:
I'm sorry I'm really unexperienced with this. I'm using `?w=1`, so whitespace is not an issue. However, I'm rather viewing file by file, which might then be several patches as once, right? Where can I find links for patch-wise changes? Thanks for bearing with me!

Reply:
https://github.com/svg-sprite/svg-sprite/pull/436/commits and open each patch to a new tab. There's also a cog button to hide whitespace there :)

Reply:
Thanks! So it's commit-wise — I thought there was a difference to patches. 😉 I'll have to take a break now (and do what my clients expect me to do on a Sunday 😥) but I'll come back later today / tonight!